src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -48,6 +48,11 @@ public static bool IsPlaceholderModeEnabled(this IEnvironment environment)
             return environment.GetEnvironmentVariable(AzureWebsitePlaceholderMode) == "1";
         }
 
+        public static bool IsRuntimeScaleMonitoringEnabled(this IEnvironment environment)
+        {
+            return environment.GetEnvironmentVariable(EnvironmentSettingNames.FunctionsRuntimeScaleMonitoringEnabled) == "1";
```

Comment:
This is surfaced to the runtime as an app setting, but it's actually managed via ARM SiteConfig - the app setting is a "system setting" not directly modifiable via app settings APIs by the user (similar to EasyAuth and other settings).